pingdom/README.md

```diff
@@ -0,0 +1,7 @@
+# Pingdom data collector
+
+It collects the data from Pingdom
+
+- **Usage:** -
```

Comment:
Use `##` for the Usage header and use triple backticks for the python command (not bold).